### ISSUE TITLE
Add support for TMT-32 and 35-plex?

### ISSUE THREAD

**Issue by jpfeuffer** [OpenMS/OpenMS#7629]
Fresh from HUPO

https://www.thermofisher.com/nl/en/home/life-science/protein-biology/protein-mass-spectrometry-analysis/protein-quantitation-mass-spectrometry/tandem-mass-tag-systems.html

Just add the following table to a new class (with .h and .cpp file) called TMTThirtyTwoPlexQuantitationMethod based on e.g. TMTEighteenPlexQuantitationMethod:
https://assets.thermofisher.com/TFS-Assets/LSG/certificate/Certificate-of-Analysis/A40000817_ZL403757.PDF

**Comment by jpfeuffer** [comment]
They seem to be combined sets of older labels and deuterated versions of them.

**Comment by FreezB11** [comment]
@jpfeuffer 
umm sir how do i get the isobaric channel information for the tmt32 plex and tmt 35 plex

**Comment by jpfeuffer** [comment]
From Thermos website (product sheet PDF).

**Comment by jpfeuffer** [comment]
I think a little knowledge about mass spectrometry is needed here. Unless @cbielow  has a clear path forward.
There might be some thinking involved. Not sure if thermo provides a single channel correction for 32. Probably you need to combine the ones from 16 and 18 or something.

**Comment by FreezB11** [comment]
@jpfeuffer so i googled few stuff up and found this 
https://pwilmart.github.io/blog/2022/08/20/TMT-channel-crosstalk
does this work it seems to be the data for 32-plex

**Comment by jpfeuffer** [comment]
Oh cool. Yes that looks like what we need. You can give it a try.

**Comment by FreezB11** [comment]
https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0018773_TMTproMassTagLabelingReagentsandKits_UG.pdf
just found this new data and i believe what you said regarding it has to do something with the 16 or 18 i believe that's what is it and sorry for troubling again but maybe if you can check this link out .

**Comment by jpfeuffer** [comment]
Yes but the second link does not include the Cross-Talk percentages.
The first link looks more complete at the first glance

**Comment by jpfeuffer** [comment]
Ah but the first link is only for 18plex

**Comment by FreezB11** [comment]
umm sir that's my doubt too coz i was going through the files and saw that the had the same data so i am not sure about it now. sorry i will look again

**Comment by jpfeuffer** [comment]
https://assets.thermofisher.com/TFS-Assets/LSG/certificate/Certificate-of-Analysis/A40000817_ZL403757.PDF

**Comment by FreezB11** [comment]
sorry sir i took a while to gather out info i stilll dont understand how to solve this issue as far as i know and looked up online i havent found any reporter ion mass the only thing i end finding again is the ion distribution

**Comment by jpfeuffer** [comment]
I think then this issue might not be the right task for you. As I said, a little mass spectrometry knowledge is required here. Thank you for trying.

**Comment by cbielow** [comment]
I think we can do without the correction for now; just analyzing the data would be a good start.

**Comment by FreezB11** [comment]
> I think we can do without the correction for now; just analyzing the data would be a good start.

Umm okay sir I shall get on to it

**Comment by FreezB11** [comment]
sir i have raised a pr for the related code for 32plex.
